actionmailbox/test/migrations_test.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require ActionMailbox::Engine.root.join("db/migrate/20180917164000_create_action_mailbox_tables.rb").to_s
+
+class ActionMailbox::MigrationsTest < ActiveSupport::TestCase
+  setup do
+    @original_verbose = ActiveRecord::Migration.verbose
+    ActiveRecord::Migration.verbose = false
+
+    @connection = ActiveRecord::Base.connection
+    @original_options = Rails.configuration.generators.options.deep_dup
+  end
+
+  teardown do
+    Rails.configuration.generators.options = @original_options
+    rerun_migration
+    ActiveRecord::Migration.verbose = @original_verbose
+  end
+
+  test "migration creates tables with default primary key type" do
+    action_mailbox_tables.each do |table|
+      assert_equal :integer, primary_key(table).type
+    end
+  end
+
+  test "migration creates tables with configured primary key type" do
+    Rails.configuration.generators do |g|
+      g.orm :active_record, primary_key_type: :string
+    end
+
+    rerun_migration
+
+    action_mailbox_tables.each do |table|
+      assert_equal :string, primary_key(table).type
+    end
+  end
+
+  private
+    def rerun_migration
+      CreateActionMailboxTables.migrate(:down)
+      CreateActionMailboxTables.migrate(:up)
+    end
+
+    def action_mailbox_tables
+      [:action_mailbox_inbound_emails]
+    end
+
+    def primary_key(table)
+      puts "TEEHEE"
```

Comment:
```suggestion
```

Reply:
Ha, this is embarrassing. Apologies for not taking out my debug comments!

Reply:
No worries, we've all done it 😄